### PR TITLE
PS-6059 Update installation doc for tarball

### PR DESCRIPTION
**Pull request by patrickbirch**
Updated installation doc with tarball information based on percona.com/downloads

### CODE REVIEW

File: doc/source/installation.rst
Comment (dutow) [inline doc/source/installation.rst]: Why the extra linebreak?

Comment (patrickbirch) [reply doc/source/installation.rst]: fixed

**Review by dutow — APPROVED** [review]

**Comment by patrickbirch** [comment]
waiting for approval or comments

**Review by tplavcic — APPROVED** [review]
LGTM

**Pull request merged**
